src/DurableTask/LogHelpers.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DurableTask;
+
+/// <summary>
+/// Non-generated helpers for <see cref="ILogger" />.
+/// </summary>
+static class LogHelpers
+{
+    public static void PurgingInstances(this ILogger logger, PurgeInstancesFilter filter)
```

Comment:
Do we need a separate static class for this or can we put it in Logs.cs?

Reply:
I can try again, but putting it in `Logs.cs` broke the code gen. Not sure if that was just my editor going haywire or not though.